### ISSUE TITLE
You can make massive custom EMP burgers

### ISSUE THREAD

**Issue by dragomagol** [tgstation/tgstation#63898]
Reporting client version: 514.1575

## Round ID:
[176159](https://scrubby.melonmesa.com/round/176159)

## Testmerges:
- [Revert "Require Node 16"](https://www.github.com/tgstation/tgstation/pull/63331)
- [lowers kilo's bombcap multiplier](https://www.github.com/tgstation/tgstation/pull/63750)
- [makes hud images only show to mobs on the same z level to save on maptick](https://www.github.com/tgstation/tgstation/pull/63558)

## Reproduction:
You can make a massive chemical reaction by combining two custom burgers full of reagents. Deepfry one burger full of stuff, put it in the new burger, they mix and create a reaction. Possible oversight by #63265

**Comment by tralezab** [comment]
oranges knew this would happen

**Comment by MrMelbert** [comment]
![image](https://user-images.githubusercontent.com/51863163/148503408-43bb7c02-68b4-4512-92e7-9e8eed381461.png)

**Comment by TemporalOroboros** [comment]
Well that was remarkably fast.

**Comment by TheBonded** [comment]
this was called out in the pr as well, I feel like it'll happen a few times during the first 2 weeks across all the servers and then everyone who did it will be either banned, leave due to everyone hating them, or get tired of making giant burgers that make large emps or explosions.

**Comment by timothymtorres** [comment]
> oranges knew this would happen

@tralezab not sure why this is marked as oversight since this was discussed as a possible outcome.  This should be considered `Not a Bug` since this behavior is intended.

**Comment by ATH1909** [comment]
also, for a massive emplosion, you'd need to actually find a way to get that much uranium

**Comment by TheBonded** [comment]
> also, for a massive emplosion, you'd need to actually find a way to get that much uranium

@ATH1909 glowberries have 25% uranium generation, you can move that onto a densified omegaweed for 210u uranium per omegaweed right off the bat

**Comment by timothymtorres** [comment]
Maybe we need to make omegaweed harder to grow?

**Comment by TemporalOroboros** [comment]
> > also, for a massive emplosion, you'd need to actually find a way to get that much uranium
> 
> @ATH1909 glowberries have 25% uranium generation, you can move that onto a densified omegaweed for 210u uranium per omegaweed right off the bat

Isn't that percentage of potency, not percentage of maximum volume? It should be limited to 25u per product.

**Comment by TemporalOroboros** [comment]
Apparently Arcane changed that at some point. Strange.

**Comment by TheBonded** [comment]
> > > also, for a massive emplosion, you'd need to actually find a way to get that much uranium
> > 
> > 
> > @ATH1909 glowberries have 25% uranium generation, you can move that onto a densified omegaweed for 210u uranium per omegaweed right off the bat
> 
> Isn't that percentage of potency, not percentage of maximum volume? It should be limited to 25u per product.

@ATH1909 no, it's percentage of volume. Almost every plant maxes out at 100u total capacity (or 200u with densified), but omegaweed has a max of 840u with densified.

Source: I made the burger that sparked this issue report.

**Comment by LemonInTheDark** [comment]
> Well that was remarkably fast.
> 
> Honestly, I'm not that worried at the moment. If it becomes endemic I'll figure out a way to limit it, but if it manages to stay relatively niche... eh.

@TemporalOroboros Fix It.

**Comment by TheBonded** [comment]
@LemonInTheDark I'm the one that originally abused this, I'll fix it. I'll go set up a cap on maximum emp explosion size, or something along those lines.

**Comment by itseasytosee** [comment]
Chem explosions don't have a max cap, therefore you can delete the entire station with a burger full of burgers full of ingredients full of thermite
See:
https://www.youtube.com/watch?v=y5zJTj4SP1U&ab_channel=TheAntropod

**Comment by TheBonded** [comment]
@itseasytosee this has already been brought up in the pr that added this, that very same video too

**Comment by itseasytosee** [comment]
to quote the creator of the video:

> Long story short: botany. (omega weed with densified chemicals + RNG (high instability plants) nitroglycerin, 1000 leaves of weed in 10 burgers, deep-fry those burgers, stick into final burger, add pyrosium, add oxygen to ignite)
